### ISSUE TITLE
Implement AND &&, OR || and semicolon ; operators

### ISSUE THREAD

**Issue by keli5** [keli5/kcsh#8]
This one will likely require a full refactoring of the command system. 

Logical AND &&: The second command will only be executed if the first command succeeds (exit status zero.)
Logical OR ||: The second command will only be executed if the first command fails (exit status non-zero.)
Semicolon ;  :  The second command will be run after the first command regardless of success or failure.

**Comment by sadanslargehole** [comment]
mmm
here we go agian

**Comment by keli5** [comment]
Completed in #15
